### DISCUSSION TITLE
Php Unit 11

### DISCUSSION THREAD

**Discussion by thomaspicdrop** [pestphp/pest#1181] [General]
Hello Pest Team,

I saw in composer.json that Pest doesn't work with PHP Unit greater than 10.5.17. Is it planned to work with PHP unit 11.*?

Greats,

Thomas

**Comment by nunomaduro** [comment]
Pest 3 will ship this summer, and it uses PHPUnit 11.

**Reply by thomaspicdrop** [reply]
Great news.

Thanks a lot!
